CONTRIBUTING.rst

```diff
@@ -140,42 +138,55 @@ Last thing
 How to make a new release
 -------------------------
 
-If you want to publish a new release, use the following steps
-
-.. code-block:: bash
-
-    # Grab the sources and install the dev tools
-    git clone https://github.com/ics-py/ics-py.git && cd ics-py
-    pip install tox poetry bump2version --user
-
-    # Make sure all the test run
-    tox && echo "Ready to make a new release" \
-        || echo "Please fix all the tests first"
-
-    # Bump the version and make a "0.8.0-dev -> 0.8.0 (release)" commit
-    bump2version --verbose release
-    # Build the package
-    poetry build
-    # Ensure that the version numbers are consistent
-    tox --recreate
-    # Check changelog and amend if necessary
-    vi CHANGELOG.rst && git commit -i CHANGELOG.rst --amend
-    # Publish to GitHub
-    git push && git push --tags
-    # Publish to PyPi
-    poetry publish
-
-    # Bump the version again to start development of next version
-    bump2version --verbose minor # 0.8.0 (release) -> 0.9.0-dev
-    # Start new changelog
-    vi CHANGELOG.rst && git commit -i CHANGELOG.rst --amend
-    # Publish to GitHub
-    git push && git push --tags
-
-Please note that bump2version directly makes a commit with the new version if you don't
-pass ``--no-commit`` or ``--dry-run``,
-but that's no problem as you can easily amend any changes you want to make.
-Further things to check:
+* Prepare environment
+
+.. code-block:: bash
+   git clone https://github.com/ics-py/ics-py.git
+   cd ics-py
+   pip install hatch
+
+* Run tests
+
+.. code-block:: bash
+
+   hatch run tox && echo "Ready to make a new release" || echo "Please fix all the tests first"
+
+* Set tag with v*
+
+.. code-block:: bash
+
+   git tag -a v8.0 -m "new release"
+
+* Build the package
+
+.. code-block:: bash
+
+   hatch build
+
+* Check changelog and amend if necessary
+
+.. code-block:: bash
+
+   vi CHANGELOG.rst && git commit -i CHANGELOG.rst --amend
+
+* Publish
+
+.. code-block:: bash
+
+   git push && git push --tags
+   hatch publish
```

Comment:
Right now this is correct but when #274 is merged, github will take care of the build and publish steps

Reply:
But lets remove it then, since the entire contribution page has to be released together with #274. https://github.com/ics-py/ics-py/pull/357/commits/574f7412545da82c3063f6f0b433749a0993e70b
Honestly, I don't quite understand the process since I can't test it but this is what I understood.